influxdb/client.py

```diff
@@ -195,21 +195,40 @@ def query(self,
               query,
               params={},
               expected_response_code=200,
+              time_precision='s',
```

Comment:
There is no time precision in queries.

see: http://influxdb.com/docs/v0.9/concepts/reading_and_writing_data.html

> The format of the returned timestamps complies with RFC3339, and has nanosecond precision